docs/reference/api.md

```diff
@@ -0,0 +1,77 @@
+# API
+
+Pebble exposes an API to allow remote clients to interact with the daemon. The API can start and stop services, add configuration layers to the plan, and so on.
+
+## Accessing the API
+
+The API uses HTTP over a Unix socket, with access to the API controlled by user ID. If `pebble run` is started with the `--http <address>` option, Pebble exposes a limited set of open-access API endpoints using the given TCP address. See {ref}`api-access-levels`.
+
+See below for some examples of how to use the API. For more examples, see "How to use the Pebble API". <!-- [David] Link to the how-to guide -->
```

Comment:
I think we should drop this paragraph (below is also the API reference), and just add a link to the how-to guide under examples when that's ready.